server/bingo/models.py

```diff
@@ -1,3 +1,77 @@
-from django.db import models # noqa
+from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
 
-# Create your models here.
+
+class UserManager(BaseUserManager):
+
+    def create_user(self, username, email, password=None):
```

Comment:
This method (and the ``create_superuser`` method) should also take in keyword arguments for the other fields on the model. This will allow creating user instances with the necessary fields filled out through this method. So the method signature should be something like ``create_user(self, username, email, password, **extra_fields)``.

Was there any reason for having the default value of ``password`` being ``None`` btw? If not, I think that default value should be removed.